windows/src/version.txt

```diff
@@ -1 +1 @@
-PRODUCTVERSION 11,0,1500,0
+PRODUCTVERSION 12,0,2000,0
```

Comment:
This was missed in the 12.0 changeover originally. Not great... Need to automate getting this from resources/version.md.